hash/snes.xml

```diff
@@ -48995,6 +49292,80 @@ to ensure nothing has been touched in the Retro Quest cart production in 2013/20
 		</part>
 	</software>
 
+	<software name="mazechal">
+		<description>MazezaM Challenge DX (v2.1, NTSC)</description>
+		<year>2020</year>
+		<publisher>Portabledev</publisher>
+		<info name="developer" value="Alekmaul" />
+		<info name="language" value="English/French" />
+		<info name="version" value="v2.1" />
+		<part name="cart" interface="snes_cart">
+			<feature name="slot" value="lorom" />
+			<dataarea name="rom" size="524288">
+				<rom name="MazezaMSnes200613_ntsc.sfc" size="524288" crc="f5a916c4" sha1="dfcc31b9979f93f82137abdf7bd4f3349eb49782" offset="0x000000" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mazechal_a" cloneof="mazechal">
+		<description>MazezaM Challenge DX (v2.1, PAL)</description>
```

Comment:
Please use mnemonic short name suffixes where possible – you could use `mazechalpal` or something here.